### PR TITLE
Add ruby 2.4 to travis (bump 2.2 and 2.3 versions)

### PR DESCRIPTION
**Pull request by jrafanie**
Add ruby 2.4 and bump the 2.2 and 2.3 versions tested in travis

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -9,6 +9,7 @@ rvm:
   - 2.1.10
   - 2.2.5
```
Comment (pboling) [inline .travis.yml]: @matthewrudy might as well bump others to latest patches, 2.2.7 and 2.3.4.  They are available as binary downloads on travis via RVM so won't affect the build speed.

Comment (jrafanie) [inline .travis.yml]: Sure, I can do that in an additional commit or PR if it's acceptable.  I wanted to only add 2.4 in this PR.  I'm ok either way.  It's easy enough to change the description.

**Comment by pboling** [comment]
👍

**Comment by matthewrudy** [comment]
Done...
thanks!

**Pull request merged**